src/generate/index.js

```diff
@@ -36,10 +36,26 @@ function injectListBetweenTags(newContent) {
 }
 
 function formatLine(contributors) {
-  return `<td>${contributors.join('</td><td>')}</td>`
+  return `<td style="text-align:center;">${contributors.join('</td><td>')}</td>`
```

Comment:
Since GitHub's markdown renderer strips out both `<style>` tags and `style` attributes, a better way would be to use `align="center".